README.md

```diff
@@ -33,12 +34,50 @@ RAPPS also accepts neutral language codes, meaning that you can do things like t
 You can also define an entry without English fallback to make it visible to certain users only.
 For instance; software from 1C, which is mostly for Russian speakers and unusable for anyone else.
 
+**__ There are some user that can speak multiple languages, for example an English-native user (with the OS set to English)  could also speak German (and use German-only software, but in case couldn't see it listed). Maybe there could be a way for the user to set which languages he would like to be listes? __**
+
+The official list of downloadable programs is kept on a public ReactOS server and synced every time RAPPS is launched for the first time.
+
+### Minimal example
+
 For a complete file format overview see the [File Schema on the ReactOS wiki](https://reactos.org/wiki/RAPPS#File_Schema)
 
-The mandatory fields are: *Name*, *Category* and *URLDownload*
-All other fields are completely optional and can be skipped.
+The mandatory fields are: *Name*, *Category* and *URLDownload*. All other fields are completely optional and can be skipped.
+
+     [Section]
+     Name = Example Program
+     Category = 1; See below for available categories
+     URLDownload = example.com/myfile.exe
+
+## FILE SCHEMA FIELD LIST
+* Name (mandatory)
+* Version
+* License
+* Description
+* SizeBytes
+* Category (mandatory)
+* URLSite
+* URLDownload (mandatory)
+* ScreenshotN
+* Icon
+
+### Name
+This is the program name that is displayed.
+
+### Version
+This is the version of the program.  **__ As it is used to check for updates, which decimal separator it uses? Comma or dot? Does it use the OS language one? Or is it fixed? __**
+
+### License
+The license that the program is based on. Typical options are : Trial, Demo, Freeware, Open-Source. Or more specifically, GPL, MIT and so on. **__ What happens if it used together with LicenseInfo? Does it also support links? __**
+
+### Description
+The description of the program giving some additional background information about what it does.
+
+### SizeBytes
+The actual size of the download in bytes. Used to display size value in the application info as well as while downloading. 
```

Comment:
I don't see the added value in duplicating the information from the wiki, that means updating even more documentation (which is already hardly ever done, so surely adding more work is not gonna help)

Reply:
I don't want to duplicate the wiki, I want to update it. The problem is that I have some questions  to ask and it is not possible to open issues on this repository (otherwise I would just have opened one). 

All the parts marked as `**__ {text} __**`, are the questions